pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -4,11 +4,13 @@
 from datetime import datetime, time
 import locale
 
+# https://github.com/python/typeshed/pull/XXXX
+# error: Module 'pytz' has no attribute 'FixedOffset'
```

Comment:
this is annoying. isort doesn't keep the comment next to the ignore

Reply:
Should we try to fix upstream in isort?